js/main.js

```diff
@@ -25,4 +35,71 @@ function handleSubmit(event) {
   data.entries.unshift(entryValues);
   $img.setAttribute('src', './images/placeholder-image-square.jpg');
   $formSubmit.reset();
+  $ul.prepend(renderEntry(entryValues));
+  toggleNoEntries();
+  viewSwap('entries');
+}
+
+function renderEntry(entry) {
+  const $li = document.createElement('li');
+  const $row = document.createElement('div');
+  const $columnHalfImg = document.createElement('div');
+  const $imgEntry = document.createElement('img');
+  const $columnHalfText = document.createElement('div');
+  const $flexColumn = document.createElement('div');
+  const $h3 = document.createElement('h3');
+  const $p = document.createElement('p');
+
+  $row.className = 'row';
+  $columnHalfImg.className = 'column-half padding-zero';
+  $imgEntry.className = 'column-full padding-zero entry-img';
+  $imgEntry.setAttribute('src', entry.image);
+  $columnHalfText.className = 'coiumn-half';
+  $flexColumn.className = 'flex-direction-column';
+  $h3.innerText = entry.title;
+  $p.innerText = entry.note;
+
+  $li.appendChild($row);
+  $row.appendChild($columnHalfImg);
+  $columnHalfImg.appendChild($imgEntry);
+  $row.appendChild($columnHalfText);
+  $columnHalfText.appendChild($flexColumn);
+  $flexColumn.appendChild($h3);
+  $columnHalfText.appendChild($flexColumn);
+  $flexColumn.appendChild($p);
+
+  return $li;
+}
+
+function handleContent(event) {
+  toggleNoEntries();
+  viewSwap(data.view);
+  for (let i = 0; i < data.entries.length; i++) {
+    $ul.appendChild(renderEntry(data.entries[i]));
+  }
+}
+
+function toggleNoEntries() {
+  const $h1 = document.createElement('h1');
+  if (data.entries.length === 0) {
+    $h1.innerText = 'THERE ARE NO ENTRIES RECORDED';
+    $ul.appendChild($h1);
+  } else {
+    const existing = $ul.querySelector('h1');
+    if (existing) {
+      existing.remove();
+    }
+  }
+}
```

Comment:
This text should be in your HTML file. The only purpose of this function is to check the length of `data.entries` and either show or hide the element accordingly. Not create it and delete it